src/app/core/services/title-utils.service.ts

```diff
@@ -0,0 +1,52 @@
+import {Injectable} from '@angular/core';
+import {Title} from '@angular/platform-browser';
+import {
+  ActivatedRoute,
+  ActivatedRouteSnapshot,
+  NavigationEnd,
+  Router,
+} from '@angular/router';
+import {concat, of} from 'rxjs';
+import {filter, map, shareReplay} from 'rxjs/operators';
+
+@Injectable()
+export class TitleUtilsService {
```

Comment:
There is now a Title Strategy, which is the native feature on how to solve this issue